pkg/virt-handler/vm.go

```diff
@@ -111,6 +111,12 @@ type netstat interface {
 	CachePodInterfaceVolatileData(vmi *v1.VirtualMachineInstance, ifaceName string, data *netcache.PodIfaceCacheData)
 }
 
+type genericMetricsManager interface {
+	Run(stopCh chan struct{})
+	StartMetricsServer(vmi *v1.VirtualMachineInstance) error
```

Comment:
nit: Rename to `StartServer` and `StopServer`?

Reply:
Agreed